lib/conduit_web/templates/layout/app.html.eex

```diff
@@ -2,33 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
     <meta name="author" content="">
+    <link rel="icon" href="/favicon.ico">
+
+    <title>Conduit CI</title>
 
-    <title>Hello Conduit!</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
   </head>
 
   <body>
+
+    <nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
+      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <a class="navbar-brand" href="">
+        <img height="19px" src="<%= static_path(@conn, "/images/conduit-white.svg") %>" alt="Conduit CI"/>
+      </a>
+
+      <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item<%#= active_class(@conn, pipeline_path(@conn, :index))%>">
+            <a class="nav-link" href="<%#= pipeline_path(@conn, :index) %>">Pipelines</a>
+          </li>
```

Comment:
Maybe it would be useful to create a helper that combines the active and the `a` generation?

Reply:
personally I think thats overkill though, as we still want to see the markup in the template more than the views

Reply:
Fair enough, is too early for things like that anyway.